### ISSUE TITLE
[A11Y] Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by israelcefrin** [pkp/immersion#103]
## Issue Description
Issue published date doesn't have minimum color contrast.

## How to reproduce / Current behavior
1) Access Archives page
2) Check the published date of any issue
3) See the contrast ratio error

## What application are you using?
 OJS version 3.4

## Stack used 
- Device: Laptop
- OS: MacOS
- Browser: Chrome
- Version: 131

## Testing method
- Automated test: BrowserStack & WAVE plugin

## Action
Increase the contrast between the foreground(text) colour and the background colour to reach the minimum recommended of 4.5:1 (currently it is only 2.35:1)

## Additional information

![Image](https://github.com/user-attachments/assets/ddb3791c-1906-41c7-9b24-1c73de099587)

**Comment by israelcefrin** [comment]
Hi @kaitlinnewson ,
Yes, they cover the same issue, feel free to close the first one.
